example/ctc/multiproc_data.py

```diff
@@ -141,4 +141,5 @@ def reset(self):
         print("Queue size on reset: {}".format(qsize))
         for i, p in enumerate(self.proc):
             p.join()
-        self.proc.clear()
+        if len(self.proc) > 0:
```

Comment:
maybe a more Pythonic way would be `if self.proc` ?

Reply:
I was using python2.7. I faced issue at that time. that is also good suggestion.